articles/cosmos-db/sql-api-sdk-java.md

```diff
@@ -39,6 +39,17 @@ The SQL API Java SDK supports synchronous operations. For asynchronous support,
 
 ## Release notes
 
+### <a name="2.1.0"/>2.1.3
+* Fixed bud in PartitionKey for Hash V2.
+
+### <a name="2.1.0"/>2.1.2
+* Added support for Composite Indexes.
+* Fixed bug in Global Endpoint Manager to force refresh.
+* Fixed bug for Upserts with pre-conditions in DirectMode.
+
+### <a name="2.1.0"/>2.1.1
+* Fixed bug in Gateway Address Cache.
```

Comment:
If possible don't capitalize first letter in each word. Can you use regular casing.

Reply:
capitalized first letter looks like the old convention. Do you want me to change it all through? Otherwise it wouldn't be consistent

Reply:
Yes please!